cmd/horologium/main.go

```diff
@@ -227,3 +243,48 @@ func sync(prowJobClient ctrlruntimeclient.Client, cfg *config.Config, cr cronCli
 	}
 	return nil
 }
+
+func shouldTriggerFailedRun(j v1.ProwJob, p config.Periodic, now time.Time, logger *logrus.Entry, previousFound bool) bool {
+	if p.RetriggerFailedRun == nil {
+		return false
+	}
+
+	runCount := 1
+	if previousFound {
+		if !j.Complete() {
+			return false
+		}
+		countLabel, exists := j.Labels[kube.ReRunLabel]
+		if exists {
+			if count, err := strconv.Atoi(countLabel); err == nil {
+				runCount = count + 1
+			}
+		}
+	}
+	if runCount > p.RetriggerFailedRun.Attempts {
```

Comment:
So this means given a vanilla-triggered job has a label with `1`, then when I configure `attempts: 1`, I **do not** actually get a retry behavior? That is a bit confusing to me, I think I would expect to get a single retry after failure for such config.

Reply:
You are right. I'll start counting from 0.